src/app/index.tsx

```diff
@@ -3,27 +3,41 @@ import React from 'react';
 import { StatusBar } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { RootStackParamList } from '@interfaces/stack';
 import ROUTES, { ROUTES_TITLE } from '@constants/routes';
 import { DEFAULT_NAV_OPTIONS } from '@constants/navigation';
+import { DEFAULT_TAB_OPTIONS } from '@constants/tabBar';
+import TabBarIcon from '@components/TabBarIcon';
 
 import Library from './screens/Library';
 import BookDetail from './screens/BookDetail';
+import WishList from './screens/WishList';
 
 const Stack = createStackNavigator<RootStackParamList>();
+const Tab = createBottomTabNavigator();
+
+function LibraryStack() {
```

Comment:
For the training is ok but, just as a comment, we keep all the navigation logic in a separated component (@components/AppNavigator) and all the navigation configuration in `@config/navigation` but great work with this 👏